modules/@apostrophecms/area/index.js

```diff
@@ -89,6 +100,14 @@ module.exports = {
   },
   methods(self) {
     return {
+      addExpandedMenu() {
+        self.apos.modal.add(
```

Comment:
Not necessary — only used when a modal is going to be instantiated via a bus event, like the manager modals.